stable/kong/values.yaml

```diff
@@ -387,3 +392,7 @@ ingressController:
     name:
 
   ingressClass: kong
+
+  podDisruptionBudget:
+    enabled: true
```

Comment:
Please set the default to `false` for both deployments.

Reply:
done!